.github/workflows/ci.yml

```diff
@@ -8,6 +8,46 @@ on:
 
 jobs:
 
+  bandit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install --upgrade bandit
+      - run: bandit -r hijack -x hijack/tests
+
+  black:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install black
+      - run: black --check --diff .
+
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v1
+      - uses: actions/checkout@v2
+      - run: python -m pip install flake8
```

Comment:
How about add
```flake8-broken-line
flake8-comprehensions
flake8-docstrings
flake8-logging-format
flake8-raise
```

Reply:
Sure why no, but I would do that in separate commits. And maybe a separate PR as well, to this already pretty big, but everything was auto-formatted by isort and black. Those extensions will require manual changes to the code, which need to be reviewed more carefully.